contrib/demo/clusters/kind/us-west1.config

```diff
@@ -3,3 +3,6 @@ apiVersion: kind.x-k8s.io/v1alpha4
 name: us-west1
 networking:
   apiServerAddress: "127.0.0.1"
+nodes:
+- role: control-plane
+  image: kindest/node:v1.15.12
```

Comment:
Why pin to this image?

Reply:
To have the West controller be started with a Kubernetes 1.15 version instead of the latest one with is 1.20.
cf. https://kind.sigs.k8s.io/docs/user/quick-start/#setting-kubernetes-version